Dockerfile

```diff
@@ -36,18 +36,26 @@ ARG PROXY_FEATURES
 
 RUN --mount=type=cache,target=/var/lib/apt/lists \
   --mount=type=cache,target=/var/tmp \
-  apt update && apt install -y time cmake
+  apt update && \
+  apt install -y \
+    time \
+    cmake \
+    golang \
```

Comment:
Do we really need golang here?

Reply:
Ah yeah that's crazy, the boring-sys crate uses go as part of the whole build process.

We probably have to figure out a better way to handle this docker file.